### ISSUE TITLE
Uniform website build and deployment across projects

### ISSUE THREAD

**Issue by vasco-santos** [nftstorage/nft.storage#2106]
We currently maintain many projects with websites: nft.storage, web3.storage, nftstorage.link ...

We should uniform them all for two reasons:
- reduce maintenance burden of everything being setup differently
- move towards a better environment setup, where all build env vars and release process is in github actions

Part of this was implemented on https://github.com/nftstorage/nft.storage/pull/2101, but we still need to fully uniform the projects.

Requirements:
- Workflow stages: check, test, preview, changelog, release
- Build in CI and deployment with wrangler for staging and production
- Add to w3s and make available on web3.storage

Basically port https://github.com/web3-storage/web3.storage/blob/main/.github/workflows/website.yml to the other places :)

**Comment by elizabeth-griffiths** [comment]
Closing as this ticket is over 1 year old. If you still need support, please re-open this issue and provide more detail. Thanks!
